### ISSUE TITLE
Implement new loss functions

### ISSUE THREAD

**Issue by Lyubava** [Samsung/veles#14]
Need to implement loss functions: 
1. Sigmoid Cross-Entropy
2. Log Softmax
